tools/ports/__init__.py

```diff
@@ -34,13 +34,11 @@
 
 
 def load_port(name):
-  expected_attrs = ['get', 'clear', 'show']
-  port = __import__(name, globals(), level=1)
+  port = __import__(name, globals(), level=1, fromlist=[None])
```

Comment:
I will double check if I can get away without it, but I copied this from your own PR

Reply:
So I double checked and without ` fromlist=[None]` it doesn't work. Like I said I had copied this bit from your PR and I assumed you had done it for a good reason. Without it, the modules under contrib are not loaded properly. 

Displaying `port.__name__` which is the name of the module, shows that for modules under ports it works (I get for example `tools.ports.sdl2`) but for ports under `contribs` it doesn't as the name is `tools.ports.contrib`.

If I use ` fromlist=[None]`, then the name is now correct `tools.ports.contrib.glfw3`

Asking Jetbrains AI give this answer (which I clipped because it is quite long but quite informative):

> In conclusion, fromlist=[None] is a way to tell `__import__` to treat the last part of the module path as the actual module that we want to import. If the list is empty or not provided `__import__` will instead return the first module in the provided path.

Note that documentation for `__import__` recommends: _Because this function is meant for use by the Python interpreter and not for general use, it is better to use `importlib.import_module()` to programmatically import a module._